website/content/posts/2023-02-02-one-year/_index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: One year of PRQL
+date: 2023-02-02
+layout: article
+toc: true
+---
+
+A year ago,
+[a proposal for a modern SQL replacement](https://news.ycombinator.com/item?id=30060784)
+was posted to Hacker News. It immediately sparked the interest of many people
+dealing with data. The project grew a community of people who are now developing
+the language, tooling, and the idea of a modern relational language. Since then
+we've opened 1577 issues & PRs on [our main repo](https://github.com/PRQL/prql),
+submitted 4211 comments, and made 1176 commits.
+
+The number of stars is skyrocketing every time the project appears on Hacker
+News, which we believe to be an indicator that people are eager to adopt the
+language if the tooling is made accessible enough.
+
+![](FQ9QSOo.png)
+
+## Where are we?
+
+Language design & development in the last year have been focused on these areas:
+
+- small quality-of-live language features (e.g. syntax for
+  [f-strings, dates, coalesce operator](https://prql-lang.org/book/syntax.html),
+  [switch](https://github.com/PRQL/prql/issues/504)),
+
+- design of basic [transforms](https://prql-lang.org/book/transforms/index.html)
+  and their [interactions](https://github.com/PRQL/prql/issues/300),
+
+- fundamentals of how [functions](https://github.com/PRQL/prql/issues/444) and
+  pipelines are [evaluated](#define-functional-semantics).
```

Comment:
Very nice list

Reply:
I might switch the order:
* Design of basic transforms
* Fundamentals of functions/pipelines
* small quality of **life** features (not "live")